tests/alternatives/main.fmf

```diff
@@ -0,0 +1,14 @@
+summary: Alternatives integration tests
+component: alternatives
+framework: beakerlib
+contact: Jan Macku <jamacku@redhat.com>
+tier: 1
+require:
+  - gcc
+  - make
+  - beakerlib
```

Comment:
nit: It is not necessary to list `beakerlib` here, it was already specified by the `framework`.

Reply:
Thank you, I missed this.